cocos/tween/actions/action-interval.ts

```diff
@@ -29,6 +29,25 @@ import { FiniteTimeAction, Action } from './action';
 import { macro, logID, errorID } from '../../core';
 import { ActionInstant } from './action-instant';
 
+// Extra action for making a Sequence or Spawn when only adding one action to it.
+class ExtraAction extends FiniteTimeAction {
```

Comment:
How about rename to DummyAction?

Reply:
It's ok for me. Actually, the name of `ExtraAction` is come from the code in `cocos2d-x`.

Reply:
It seems DummyAction is more meaningful as it does nothing, just make the tween system happy?

Reply:
Renamed.